packages/ra-core/src/controller/field/ReferenceManyFieldController.spec.tsx

```diff
@@ -7,6 +7,76 @@ import renderWithRedux from '../../util/renderWithRedux';
 
 describe('<ReferenceManyFieldController />', () => {
     it('should set loaded to false when related records are not yet fetched', async () => {
+        const ComponentToTest = ({ loaded }: { loaded?: boolean }) => {
+            return <div>loaded: {loaded.toString()}</div>;
+        };
+
+        const { queryByText } = renderWithRedux(
+            <ReferenceManyFieldController
+                resource="foo"
+                source="items"
+                reference="bar"
+                target="foo_id"
+                basePath=""
+            >
+                {props => <ComponentToTest {...props} />}
+            </ReferenceManyFieldController>,
+            {
+                admin: {
+                    references: {
+                        oneToMany: {},
```

Comment:
this is not the normal shape of the state. there is a key representing the reference between `reference` and `oneToMany`